core/src/main/java/org/mule/runtime/core/api/processor/MessageProcessors.java

```
@@ -96,7 +96,17 @@ public static MessageProcessorChain newExplicitChain(List<Processor> processors)
 
   public static Event processToApply(Event event, Processor processor) throws MuleException {
     try {
-      return just(event).transform(processor).otherwise(Exceptions.EventDroppedException.class, ede -> empty()).block();
+      return just(event).transform(processor).otherwise(EventDroppedException.class, mde -> empty()).block();
+    } catch (Throwable e) {
+      throw rxExceptionToMuleException(e);
+    }
+  }
+
+  public static Event processToApply(Event event, Flow flow) throws MuleException {
+    try {
+      just(event).transform(flow).doOnError(throwable -> {
```

isn't this just ignoring the error?